### PR TITLE
Revert "Update ci.yml to be dispatchable and skippable"

### PR DESCRIPTION
**Pull request by adimoldovan**
Reverts woocommerce/woocommerce#45331

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -302,7 +236,7 @@ jobs:
         'project-lint-jobs',
         'project-test-jobs',
       ]
-    if: ${{ always() && github.event_name != 'pull_request' && ! github.event.pull_request.head.repo.fork }}
+    if: ${{ always() && github.event_name != 'pull_request' && ! github.event.pull_request.head.repo.fork  }}
```
Comment (kalessil) [inline .github/workflows/ci.yml:239]: Doesn't look like properly reverted.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -339,7 +273,7 @@ jobs:
         'project-jobs',
         'project-test-jobs',
       ]
-    if: ${{ always() && needs.project-jobs.outputs.report-jobs != '[]' && ! github.event.pull_request.head.repo.fork }}
+    if: ${{ always() && needs.project-jobs.outputs.report-jobs != '[]' && ! github.event.pull_request.head.repo.fork  }}
```
Comment (kalessil) [inline .github/workflows/ci.yml:276]: Doesn't look like properly reverted.

**Review by kalessil — APPROVED** [review]
Noted some diviations from original changes - see comments. Pre-approving.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (adimoldovan) [reply .github/workflows/ci.yml:239]: I updated this on purpose, to keep the condition of only executing the step on non-forks.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (adimoldovan) [reply .github/workflows/ci.yml:276]: Same as above, updated it on purpose.

**Pull request merged**